bootstrap/src/main/java/jenkins/bootstrap/Plugin.java

```diff
@@ -75,19 +75,27 @@ void process(DependenciesTxt core, List<URL> jars, List<DependenciesTxt> overrid
         explode();
 
         File dependencies = new File(destDir,"META-INF/dependencies.txt");
+        boolean hasDependenciesNewerThanCore = false;
         try (InputStream i = new FileInputStream(dependencies)) {
             DependenciesTxt deps = new DependenciesTxt(i);
             for (Dependency d : deps.dependencies) {
+                if (!core.hasNewerThan(d)) {
```

Comment:
This condition seems wrong.

Reply:
it's fine: we are checking plugin do offer an update vs core, so core don't have a newer version

Reply:
Well, `<` isn't the complement of `>`, it's the complement of `≥`. Either this logic is wrong or the names are wrong.

Reply:
right indeed, fixed by daabdabe7285c43b28962b147797468bbc090441